internal/cli/get_test.go

```diff
@@ -185,3 +187,343 @@ func TestHandleListBatching(t *testing.T) {
 		})
 	}
 }
+
+func TestParseAndValidateKindNameFromArgs(t *testing.T) {
+	tests := []struct {
+		name          string
+		args          []string
+		expectedKind  string
+		expectedNames []string
+		expectError   bool
+		errorContains string
+	}{
+		// TYPE/NAME format tests
+		{
+			name:          "single_device_slash_format",
+			args:          []string{"device/test1"},
+			expectedKind:  DeviceKind,
+			expectedNames: []string{"test1"},
+			expectError:   false,
+		},
+		{
+			name:          "short_form_slash_format",
+			args:          []string{"dev/test1"},
+			expectedKind:  DeviceKind,
+			expectedNames: []string{"test1"},
+			expectError:   false,
+		},
+		{
+			name:          "plural_slash_format",
+			args:          []string{"devices/test1"},
+			expectedKind:  DeviceKind,
+			expectedNames: []string{"test1"},
+			expectError:   false,
+		},
+		{
+			name:          "enrollment_request_slash_format",
+			args:          []string{"er/test1"},
+			expectedKind:  EnrollmentRequestKind,
+			expectedNames: []string{"test1"},
+			expectError:   false,
+		},
+		{
+			name:          "slash_format_no_name",
+			args:          []string{"device/"},
+			expectedKind:  DeviceKind,
+			expectedNames: []string{},
+			expectError:   false,
+		},
```

Comment:
Seems a bit surprising that this works and does not return an error?

Reply:
fixed